test/integration/consul-container/test/jwtauth/jwt_auth_test.go

```diff
@@ -92,25 +120,25 @@ func createServices(t *testing.T, cluster *libcluster.Cluster) libservice.Servic
 		HTTPPort: 8080,
 		GRPCPort: 8079,
 	}
+	apiOpts := &api.QueryOptions{Token: cluster.TokenBootstrap}
```

Comment:
Noob question: What's `TokenBootstrap`? Should this  value be passed in as a parameter for tests that require it?

Reply:
this is the initial management token (the first token with unrestricted privileges). It is a result of `consul acl bootstrap` command usually. 

What's an example of a test you think it should be passed to? i think this is fine as everything has access to the cluster anyways unless i am misunderstanding the question